src/AppBundle/Controller/Admin/UserController.php

```diff
@@ -38,8 +38,9 @@ public function viewAction(Request $request, User $user)
             new AdminUserType(),
             $user,
             [
-                'manager' => $this->getDoctrine()->getManager(),
-                'locale' => $request->getLocale(),
+                'manager'       => $this->getDoctrine()->getManager(),
+                'locale'        => $request->getLocale(),
+                'wantToLearn'   => (int)$user->getWantToLearn()
```

Comment:
You also might want to tell Symfony developers not to use casting https://github.com/symfony/symfony/blob/2.7/src/Symfony/Component/Form/Form.php#L348